process/transactionProcessor.go

```diff
@@ -30,16 +30,17 @@ const TransactionSimulatePath = "/transaction/simulate"
 // MultipleTransactionsPath defines the multiple transactions send path of the node
 const MultipleTransactionsPath = "/transaction/send-multiple"
 
-// UnknownStatusTx defines the response that should be received from an observer when transaction status is unknown
-const UnknownStatusTx = "unknown"
-
 const (
-	withResultsParam    = "?withResults=true"
-	checkSignatureFalse = "?checkSignature=false"
-	bySenderParam       = "&by-sender="
-	fieldsParam         = "?fields="
-	lastNonceParam      = "?last-nonce=true"
-	nonceGapsParam      = "?nonce-gaps=true"
+	withResultsParam      = "?withResults=true"
+	checkSignatureFalse   = "?checkSignature=false"
+	bySenderParam         = "&by-sender="
+	fieldsParam           = "?fields="
+	lastNonceParam        = "?last-nonce=true"
+	nonceGapsParam        = "?nonce-gaps=true"
+	txCompletedEvent      = "completedTxEvent"
```

Comment:
can you use these constants from here https://github.com/multiversx/mx-chain-core-go/blob/16023cace5b78df86fbaa1e6a6b3778873f58c25/core/constants.go#L250 
for the logs identifiers

Reply:
also, take into consideration the log with the identifier `internalVMErrors`

Reply:
great, will do

Reply:
done